doc/manual/src/projects.md

```diff
@@ -458,6 +458,42 @@ Hydra can send email notifications when the status of a build changes.
 This provides immediate feedback to maintainers or committers when a
 change causes build failures.
 
+The feature can be turned on by adding the following line to `hydra.conf`
+
+``` conf
+email_notification = 1
+```
+
+By default, Hydra only sends email notifications if a previously successful
+build starts to fail. In order to force Hydra to send an email for each build
+(including e.g. successful or cancelled ones), the environment variable
+`HYDRA_FORCE_SEND_MAIL` can be declared:
+
+``` nix
+services.hydra-dev.extraEnv.HYDRA_FORCE_SEND_MAIL = "1";
+```
+
+SASL Authentication for the email address that's used to send notifications
+can be configured like this:
```

Comment:
Is there somewhere the user can learn more about these variables, and perhaps other similar variables?

Reply:
I have to admit that I looked up the `sasl_username`/`sasl_password` things directly up from the source code of the CPAN module for the mailer :D 

But I guess it makes sense to leave a link to the CPAN page of the module at least :)

Reply:
Added a link to the metacpan page.